### DISCUSSION TITLE
how nuxt3 import the entire router.js when migrate from vue3

### DISCUSSION THREAD

**Discussion by mebest100** [nuxt/nuxt#24633] [Questions]
I'm now trying migrate from vue 3 to nuxt3. And I now nuxt define router based on directory but you know I'm not fresh and start  with nuxt3 but make migration from vue3.  Hence how I make nuxt just import the router.js which already defined in vue3, can I config it in nuxt.config.ts?　Thx!

**Comment by TheAlexLichter** [comment]
You can use [`router.options.ts`](https://nuxt.com/docs/guide/going-further/custom-routing#using-approuteroptions) for this ☺️

**Comment by mebest100** [comment]
> You can use [`router.options.ts`](https://nuxt.com/docs/guide/going-further/custom-routing#using-approuteroptions) for this ☺️

Okey, but which part I will place the external router.js if adopt router.options as below
for example :
Import router  from '/router.js '  <== which come from vue3 
Then where  I will put the router in below config ?
export default defineNuxtConfig({
  router: {
    options: {}
  }
})
